evaluation/SWE-bench/Makefile

```diff
@@ -0,0 +1,22 @@
+DOCKER_BUILD_REGISTRY=ghcr.io
+DOCKER_BUILD_ORG=opendevin
+DOCKER_BUILD_REPO=eval-swe-bench
+DOCKER_BUILD_TAG=v0.1
+FULL_IMAGE=$(DOCKER_BUILD_REGISTRY)/$(DOCKER_BUILD_ORG)/$(DOCKER_BUILD_REPO):$(DOCKER_BUILD_TAG)
```

Comment:
In the future, it'd be great if we `git tag` this repo, the github action applies that tag to the image.

Reply:
if we have standard about this, the tag or branch to control is a good way.